test/middleware/CompanyAuthMiddleware.test.ts

```diff
@@ -156,13 +161,57 @@ describe('Company Authentication Middleware', () => {
 
         const startingWithRedirectUrl = (redirect : string) => redirect.startsWith(redirectUrl);
 
-        await companyAuthMiddleware.handler(request, response, nextFunction);
+        await companyAuthMiddleware.handler(req, response, nextFunction);
         nextFunction.didNotReceive();
         response.received(1).redirect(Arg.is(startingWithRedirectUrl));
 
     });
 });
 
+describe('All pages after the Penalty Details page:', () => {
```

Comment:
As we discussed please move it into its own test file.

Reply:
Done